src/utils/server/apiFunctions.ts

```diff
@@ -103,14 +94,23 @@ const buildShouldQueryClause = (queryString: string) => {
 };
 
 // Helper to build filter query clauses based on facets
-const buildFilterQueryClause = ({ field, value }: Facet) => {
-  let filterQueryClause = {
-    term: {
-      [`${field}.keyword`]: { value },
-    },
-  };
+const buildFilterQueryClause = (filter: Facet): QueryDslQueryContainer => {
+  if (Array.isArray(filter.value)) {
```

Comment:
In what situation would you have the filter value as an array?

Reply:
cc: @kouloumos

Reply:
An array filter means OR clause between the filters in the array. For more you can read the comment on the line below this or the extended explanation in the connected issue.